src/Jackett.Common/Indexers/Abstract/CouchPotatoTracker.cs

```diff
@@ -96,10 +96,12 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             {
                 foreach (JObject r in json["results"])
                 {
-                    var release = new ReleaseInfo();
-                    release.Title = (string)r["release_name"];
-                    release.Comments = new Uri((string)r["details_url"]);
-                    release.Link = new Uri((string)r["download_url"]);
+                    var release = new ReleaseInfo
+                    {
+                        Title = (string)r["release_name"],
+                        Comments = new Uri((string)r["details_url"]),
+                        Link = new Uri((string)r["download_url"])
+                    };
                     release.Guid = release.Link;
                     release.Imdb = ParseUtil.GetImdbID((string)r["imdb_id"]);
```

Comment:
This one stops at `release.Guid` because it uses a variable from release info. We could change this in one of two ways:

```C#
var release = new ReleaseInfo();
var link = new Uri(...); // Breaks the initializer finding, all are setters
```
or
```C#
var link = new Uri(...);
var release = new ReleaseInfo
{
//...
    Link = link,
    Guid = link,
// All values can be in the initializer
};
```

We have to make a decision on which form we want to allow, because there is zero settings that allow your preference, it's either all initializers when possible, or no initializers ever.

Reply:
The 1st looks like a hack to me. I would prefer to disable the style with comments.
```
// ReSharper disable once UseObjectOrCollectionInitializer
```
If I have to chose I prefer the 2nd but that implies to declare a lot of temporary variables (I don't much of C# but I'm sure that will take memory...) and a lot of work (there are 90 indexers in C#).
Maybe you can change object initializers later, in other PR. I agree with the other changes.

Reply:
Yeah, I'll redo this one to skip the initializers, they're in their own subcategory in the tool.